lib/src/suso.cpp

```diff
@@ -79,3 +79,33 @@ bool Sudoku::solveNakedSingles() {
     }
     return changed;
 }
+
+bool Sudoku::hasEmptyCell(position &pos) {
```

Comment:
Evtl. noch umbenennen, damit deutlicher wird, dass `pos` in der Funktion geändert wird